code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -187,7 +187,7 @@
 		if(M == summoner)
 			to_chat(M, "<span class='changeling'><i>[src]:</i> [input]</span>")
 			log_say("(GUARDIAN to [key_name(M)]) [input]", src)
-		else if(M in GLOB.dead_mob_list)
+		else if(M in GLOB.dead_mob_list && M.client && M.stat == DEAD && !isnewplayer(M))
```

Comment:
Bit of a redundant dead check, but I used the same checks here as the other ghost communication messages use. Better safe than sorry in case that list isn't entirely correct.